package.json

```diff
@@ -187,6 +190,10 @@
                         "disable-linters": {
                             "type": "array",
                             "description": "List of linters by name to disable"
+                        },
+                        "resource-context-menu": {
+                            "type": "boolean",
+                            "description": "If true, show create/apply/delete in the context menu when you right click on any YAML file"
```

Comment:
I'm not sure we need the 'is true' and I don't think we should list the specific commands because that will need to be maintained.  How about e.g. `Show Kubernetes resource commands on file context menu for all YAML files`?

Reply:
done.

Reply:
done.